dist/etc/scylla-manager.yaml

```diff
@@ -102,29 +102,27 @@ http: 127.0.0.1:5080
 # the same snapshot. If exceeded, new run with new snapshot will be created.
 # Zero means no limit.
 #  age_max: 24h
+#
+# Frequency with which Scylla Manager polls upload status.
+#  long_polling_timeout_seconds: 10
+
+# Restore service configuration.
+#restore:
+# Minimal amount of free disk space required for node to be used in restore process.
+#  disk_space_free_min_percent: 10
+#
+# Frequency with which Scylla Manager polls download status.
+#  long_polling_timeout_seconds: 10
 
 # Repair service configuration.
 #repair:
-# Frequency Scylla Manager poll Scylla node for repair command status.
-#  poll_interval: 50ms
-#
 # Maximal time a paused repair is considered fresh and can be continued,
 # if exceeded repair will start from the beginning. Zero means no limit.
 #  age_max: 0
 #
 # Specifies how long repair will wait until all ongoing repair requests finish
 # when repair task is stopped. After this time, task will be interrupted.
 #  graceful_stop_timeout: 30s
-#
-# Force usage of certain type of repair algorithm. Allowed values are row_level, legacy, and auto.
-# row_level means that Scylla Manager will use row level repair optimised algorithm.
-# legacy means that Scylla Manager will be splitting ranges to shards when making repair requests.
-# Default value is auto which means that repair type will be auto detected based on Scylla versions in the managed cluster.
-#  force_repair_type: auto
-#
-# Distribution of data among cores (shards) within a node.
-# Copy value from Scylla configuration file.
-#  murmur3_partitioner_ignore_msb_bits: 12
```

Comment:
Nice ! 
But it's a leftover from pre 3.2 release. 
Have nothing common with backup and restore pkgs separation.

Please introduce it with other PR.

`scylla-manager/pkg/service/repair/config.go` should be updated to.
https://github.com/scylladb/scylla-manager/blob/78ad309daeefdc50a711a924e4674dbe6b8dd7be/pkg/service/repair/config.go#L31-L32